Re-design

danisal opened the pull request.

ghost commented:
# [Gatsby Cloud](https://www.gatsbyjs.com) Build Report
**Crystallize Gatsby Boilerplate**

:tada: Your build was successful! [See the Deploy preview here.](https://crystallizegatsbyboilerplate-design.gtsb.io)

## Build Details

[View the build logs here.](https://gatsbyjs.com/dashboard/2fc1d72d-43a0-4601-9f3f-b551ed700a29/sites/38b65653-e88a-4e2b-b60b-a26d37c3a5fb/builds/f0ac48c9-0f8c-4aa2-88b6-5ff74af20b96/details)

:clock1: Build time: **8m**

## Performance

### Lighthouse report

| Metric | Score |
| ----- | ---- |
| Performance |   :large_orange_diamond: 79   |
| Accessibility |   :large_orange_diamond: 83   |
| Best Practices |   :green_heart: 100   |
| SEO |   :large_orange_diamond: 82   |

:link: [View full report](https://build-f0ac48c9-0f8c-4aa2-88b6-5ff74af20b96.gtsb.io/reports/lighthouse/index.html)

hakonkrogh committed:
fixed layout on frontpage

even more design from nextjs boiler

forgot to add as always

first pass on design in gatsby boiler

fixed footer height bug

added query to get grid relations, fixed magazine

hakonkrogh committed:
Small css adjustments
